### ISSUE TITLE
Conflicting Information 

### ISSUE THREAD

**Issue by ghost** [OpenMined/docs#40]
The docs currently state 2 conflicting messages regarding the return of trained models to Data Scientists.

Under Components: Sonar is said to return the model to the Data Scientist.
Sonar - A federated learning server running on the blockchain that handles all campaign requests, holding Bounty in trust. This library communicates with the Capsule to generate PGP keys and deliver the final, trained results back to the Data Scientist. It also communicates with miners, collecting Gradients and distributing Bounty accordingly.

Under Repositories: Capsule is said to return the model to the Data Scientist.
Capsule (Python)
PGP key generator, also responsible for delivering the trained and decrypted dataset back to data scientist

Which is it?

**Comment by anoff** [comment]
Good catch. Always a bad idea to have the same thing in two places. Is there a reason not to merge the definitions?

To my understanding the answer is between both those definitions. The DS requests the final model from sonar which will ask capsule to re-encrypt it with the DS pub key, store it somewhere and tell the DS via sonar.
Sonar is the interface, noone ever directly interfaces with capsule except sonar.

If that definition makes sense to you we only need to figure out which place to put it in :)
